pkgs/os-specific/linux/xone/default.nix

```diff
@@ -45,4 +36,3 @@ stdenv.mkDerivation rec {
     platforms = platforms.linux;
   };
 }
-
```

Comment:
Unrelated changes, please revert.

Reply:
Is there a recommended formatter? I have been using nixpkgs-fmt which adds the extra new line

Reply:
Your previous commit was removing the last line. That's the reason why I asked you to revert the changes.

Reply:
Sorry about that my mistake